std_msgs/Header header
string controller_state
string autopilot_state
geometry_msgs/Pose goal
geometry_msgs/Pose error
geometry_msgs/Twist cmd_manual
geometry_msgs/Twist cmd_autopilot
geometry_msgs/Vector3 pid
